string group
geometry_msgs/PointStamped point
---
kinematics_cache/IKv2[] results
